lib/config.dart

```diff
@@ -4,6 +4,8 @@ const String recordStorage = "/sdcard/Lingua";
 
 const String defaultTargetLang = "Yemba";
 
+const String defaultSourceLang = "francais";
```

Comment:
capital F in "francais"

Reply:
for uniformity, I decided to use languages in small case letters.

Reply:
`But I will display by capitalizing the word.`